base/markdown/Common/inline.jl

```diff
@@ -106,6 +106,42 @@ function footnote_link(stream::IO, md::MD)
     end
 end
 
+@trigger '<' ->
+function autolink(stream::IO, md::MD)
+    withstream(stream) do
+        startswith(stream, '<') || return
+        url = readuntil(stream, '>')
```

Comment:
Sorry, I misread it. I think you're right and need to do:

`text = readuntil(stream, '>', match = '<')`

I originally read it too quickly and thought setting it to `' '` to would stop reading at whitespace.